### PR TITLE
simple lock exit farm fix

### PR DESCRIPTION
**Pull request by psorinionut**

### CODE REVIEW

**Review by dorin-iancu — APPROVED** [review]

File: locked-asset/simple-lock/src/proxy_farm.rs [hunk 1]
```diff
@@ -238,6 +247,15 @@ pub trait ProxyFarmModule:
             );
         }
 
+        if exit_farm_result.remaining_farm_tokens.amount > 0 {
+            self.send().direct_esdt(
+                &caller,
+                &exit_farm_result.remaining_farm_tokens.token_identifier,
```
Comment (sasurobert) [inline locked-asset/simple-lock/src/proxy_farm.rs:253]: shouldn't you send proxy token here ?

File: locked-asset/simple-lock/src/proxy_farm.rs [hunk 2]
```diff
@@ -238,6 +247,15 @@ pub trait ProxyFarmModule:
...
+                exit_farm_result.remaining_farm_tokens.token_nonce,
```
Comment (dorin-iancu) [inline locked-asset/simple-lock/src/proxy_farm.rs:254]:
```suggestion
                &payment.token_identifier,
                payment.token_nonce,
```

**Review by dorin-iancu — CHANGES_REQUESTED** [review]

File: locked-asset/simple-lock/src/proxy_farm.rs (hunk 2 above)
Comment (psorinionut) [reply locked-asset/simple-lock/src/proxy_farm.rs]: Updated.

**Review by sasurobert — APPROVED** [review]

**Review by dorin-iancu — APPROVED** [review]

File: locked-asset/simple-lock/src/proxy_farm.rs (hunk 1 above)
Comment (psorinionut) [reply locked-asset/simple-lock/src/proxy_farm.rs]: Updated.

**Pull request merged**